### DISCUSSION TITLE
@capacitor/cli version returns 3.0.1 instead of 5.0.0

### DISCUSSION THREAD

**Discussion by unstopp** [ionic-team/capacitor#6571] [General]
Hello,

I have updated @capacitor/cli to latest version (tried both @latest and @next when npm installing)
I get @capacitor/cli: "^5.0.0-rc.2" as of today (7th May 2023) using "npm install @capacitor/cli@next"

However, when running "capacitor -V", I am getting version 3.0.1. I was expecting 5.0.0

Is this correct? Any feedback is much appreciated, thank you!

Best,
Marius

**Comment by jcesarmobile** [comment]
Latest is 5.0.1
Maybe you have an old Capacitor version installed globally?
What do you get if you run `npx cap version` instead of `capacitor version`?
When you run `npm install @capacitor/cli@latest` do you see any error in the console/terminal that indicates that the version couldn't be installed and it installed a different version?

**Comment by unstopp** [comment]
**npx cap version** returns: **Unknown command: version**
Did another npm install on the latest cli and I successfully get ^5.0.1 in my package.json of the ionic project

I did **npm install @capacitor/cli@latest -g** and it seems to have updated **globally to 5.0.1**.
Now "**capacitor -V**" is returning 5.0.1 and "**npm cap --version**" is returning 5.0.1 as well

I guess it's fine, thank you!

**Reply by jcesarmobile** [reply]
Don't install `@capacitor/cli` globally, it should be local to your project and that's probably why you were getting an old version, because you were checking the global one instead of the local.
